CONTRIBUTING.md

```diff
@@ -5,6 +5,13 @@ The following set of guidelines will help you in contributing to this Arcus repo
 
 But, no worries, if you forget something or have problems with some of these steps, we're happy to help! In those cases: create a PullRequest on this repository so we can talk about it.
 
+This contribution guide has the following contents:
+- [PowerShell modules](#powershell-modules)
+- [How to add a new script in an existing module](#how-to-add-a-new-script-to-an-existing-module)
```

Comment:
The link currently doesn't work, due to the difference in "**in** an existing" vs "**to** an existing".
Should be "to an existing module".
Can you adjust this in the link here and the header further down the page?

```suggestion
- [How to add a new script to an existing module](#how-to-add-a-new-script-to-an-existing-module)
```

Reply:
Aha, thanks! Missed that one. Fixed it.